src/util/generate-pr-message.test.ts

```diff
@@ -1,51 +1,128 @@
+import generate, { generatePrMessageWithFooter } from './generate-pr-message';
 import { IStepsResults } from './execute-steps';
-import generatePrMessage from './generate-pr-message';
-
-const specs = [
```

Comment:
Need to revisit.  May prefer previous test suite over new and more verbose test suite.